### DISCUSSION TITLE
Read only problem

### DISCUSSION THREAD

**Discussion by Kstro243** [planetscale/discussion#581] [Q&A]
Hello, so I'm trying to Edit a row of a table from my swagger API with .NET, but it throws an error apparently because the database doesn't work. But when I try to read certain information by a get it does work, so I'm assuming database is readonly? How do I make it so I can update rows too?

**Comment by jgreet** [comment]
Hi @Kstro243, is it possible that you're connecting with a read-only password? Perhaps you can try new credentials with the Read/Write or Admin role.

https://planetscale.com/docs/concepts/password-roles

If that's not it, it might be helpful to see the full error message that the database is returning.
